contracts/solidity/NFTXMarketplace0xZap.sol

```diff
@@ -650,6 +627,28 @@ contract NFTXMarketplace0xZap is OwnableUpgradeable, ReentrancyGuardUpgradeable,
   }
 
 
+  /**
+   * @notice Allows our zap to be paused to prevent any processing.
+   * 
+   * @param _paused New pause state
+   */
+
+  function pause(bool _paused) external onlyOwner {
+    paused = _paused;
+  }
+
+
+  /**
+   * @notice Allows our zap to set the swap target for 0x.
+   * 
+   * @param _swapTarget The new swap target to used
+   */
+
+  function setSwapTarget(address payable _swapTarget) external onlyOwner {
+    swapTarget = _swapTarget;
```

Comment:
Under what situations would this need to be changed? This means owner could steal from any approvals.

Reply:
I'll shift this to the constructor, make it immutable and update to have it factored into the deployment script 👍